src/lib/state.ts

```diff
@@ -18,13 +18,17 @@ import { IDL } from '@dfinity/candid';
 export class User {
 	public principal: Principal;
 	public accountId: string;
+	public identityProvider: 'Plug' | 'II' | 'Nfid';
```

Comment:
Can we use `NFID` instead of `Nfid`?

Reply:
Actually we do not use this field anymore. I am going to get rid of it